plugins/importexport/datacite/classes/DOIExportPlugin.inc.php

```diff
@@ -239,7 +239,7 @@ function process(&$request, &$journal) {
 					);
 					break;
 				} else { // Register selected objects.
-					assert($request->getUserVar('register'));
+					assert(!empty($request->getUserVar('register')));
```

Comment:
See http://php.net/manual/en/function.empty.php -- before PHP 5.5.0, empty() could only handle variables, not expressions.